tardis/opacities/opacity_solver.py

```diff
@@ -39,18 +43,14 @@ def solve(self, legacy_plasma) -> OpacityState:
         -------
         OpacityState
         """
-        atomic_data = legacy_plasma.atomic_data
+        atomic_data = plasma.atomic_data
```

Comment:
Get rid of atomic_data variable too here, in line with the change in opacity_state.py